typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -28,6 +34,7 @@ export interface ChainMetadata {
   /** Hyperlane domain, only required if differs from id above */
   domainId?: number;
   name: ChainName;
+  type: ProtocolType;
```

Comment:
Can we rename this to `protocol` or `protocolType`. As per https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1867, would like to harmonize `AgentChainSetupBase` and `ChainMetadata`, or was there a reason to use a different key for the same purpose value? Or @jmrossy did you call this `type` here as well?

Reply:
I called it `protocol`. +1 for renaming to that :)

Reply:
Protocol it is!